lib/ViewModels/FeatureInfoPanelViewModel.js

```diff
@@ -139,21 +139,25 @@ FeatureInfoPanelViewModel.prototype.showFeatures = function(features) {
         }
 
         var feature = features.features[0];
+
         if (!defined(feature.position)) {
             feature.position = features.pickPosition;
         }
-
-         that.terria.selectedFeature = feature;
+        that.terria.selectedFeature = feature;
         that.name = feature.name ? feature.name : feature.id;
-        if (defined(feature.description)) {
-            that.html = feature.description.getValue( that.terria.clock.currentTime);
+
+        if (defined(feature.properties)) {
+            var data = feature.properties.getValue(feature),
+                template = feature.imageryLayer.infoTemplate;
+
+            that.html = template ? readTemplate(data, template) : feature.description.getValue();
```

Comment:
Need to pass `that.terria.clock.currentTime` to `getValue` here, too, just in case the description is time-varying.